embedded.go

```diff
@@ -25,9 +32,9 @@ func initHTTPEmbedded(r *mux.Router) {
 }
 
 func handleCreateEmbedded(creq *apps.CallRequest) apps.CallResponse {
-	client := appclient.AsBot(creq.Context)
+	client := appclient.AsActingUser(creq.Context)
```

Comment:
Does this really fix the issue? Is there an underlying issue with using the bot's token here? The issue occurs even when the bot is a member of the current channel.

Reply:
Yes and no, the real issue was `.ChannelID`->`.Channel.Id` change just below. But in my case, I tried it in a channel that the bot was NOT a member of the channel, and that was an error as well.